packages/fields/src/types/Select/views/Filter.js

```diff
@@ -25,6 +25,7 @@ const SelectFilterView = ({ innerRef, field, value, onChange }) => {
   };
 
   const handleSelectChange = newValue => {
+    if (newValue.length == 0) return;
```

Comment:
```suggestion
    if (!newValue.length) return;
```
This is a little confusing, do we want to return if the length is exactly `0` vs undefined? Or are we ok to just check we have a non-empty array by checking for length? 

If just checking for a non-empty array I'd suggest we make it a non-inverted check and just wrap the next 2 lines.
A positive check is alway easier to read and since we have now checked it's an array do we need to concat or just make a copy? 
```
if (newValue.length) {
  onChange({ ...value, options: [...newValue] });
}